src/_guides/bump-sh-tutorials/2024-05-14-migrate-from-swaggerhub.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Migrate from SwaggerHub
+authors: phil
+excerpt: Move your OpenAPI out of SwaggerHub and into your own source control, so you can control more of the workflow with all the same benefits.
+---
+
+Tools like SwaggerHub provide an editing and hosting experience for OpenAPI documents, allowing teams to fork APIs, add new versions, diff, and compare, which was pretty handy back in the day. These days it can feel like recreating Git with a walled-garden, needing to be imported and exported, or joined together with "integrations" that involve manually setting up webhooks to ping your other tools when there are changes. Many teams are starting to question how useful that is.
+
+Bump.sh gives teams more flexibility in how they work, but the most common workflow assumes that your OpenAPI is your source of truth, and that source of truth lives with the source code in Git, so instead of managing source code and OpenAPI separately you have it all in the same pull requests as you go. Your documentation can still be published immediately on changes, but it all happens from your Git repository or CI/CD setup as a central point, instead of being stuck in somebody elses server.
+
+## Step 1: Export OpenAPI from SwaggerHub
+
+To liberate your OpenAPI from SwaggerHub for good, head to "My Hub", open each API one at a time, then click "Export" > "Download API" > "YAML Unresolved".
+
+![](/images/guides/migrating-from-swaggerhub/export-from-swaggerhub.png)
+
+This will download a version which still has `$ref`'s, which will keep the file size smaller and avoid duplication of components.
```

Comment:
Question: does this work if the definition has external references?

Reply:
Unresolved isn't messing with your refs so it will be fine. Exporting the resolved version could go either way.